typescript/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "scripts": {
+    "build": "tsc"
+  },
+  "files": [
+    "build"
+  ],
+  "dependencies": {
+    "axios": "^0.21.1",
+    "http-status-codes": "^2.1.4",
+    "kurtosis-core-api-lib": "^0.13.5",
```

Comment:
You shouldn't actually need this `kurtosis-core-api` dependency - it should get pulled in transitively by testsuite-api-lib

Reply:
I actually tried this by doing the following:

Instead of having `import { ServiceID } from "kurtosis-core-api-lib";`, I would try `import { ServiceID } from "kurtosis-testsuite-api-lib";`. However, this would not work as they are not accessible via kurtosis-testsuite-api-lib. I think I would explicitly need to define them in the index.ts of kurtosis-testsuite-api-lib